addon/mixins/scroller-events.js

```diff
@@ -78,11 +78,20 @@ function handleMove(e) {
   }
 }
 
+function scrollingDidStart() {
+  console.log('scrollingDidStart');
+  Ember.run(() => {
+    this.get('scrollerRegistry').startScrolling();
+  });
+}
+
 function scrollingDidComplete() {
+  console.log('scrollingDidComplete');
   Ember.run(() => {
     this.get('scrollerRegistry').endScrolling();
   });
   setTimeout(()=>{
+    // if (this.isDestroyed || this.isDestroying) { return; }
     Ember.run(() => {
       this.set('isScrolling', false);
```

Comment:
But calling `set('isScrolling', false)` has a custom setter that triggers other actions, so not sure what to do about that.